packages/gestalt/src/Shared/ToastSubcomponents.js

```diff
@@ -139,6 +139,14 @@ export function ToastImageThumbnail({
   );
 }
 
+export function BannerOverlayIconThumbnail({
+  thumbnail,
+}: {
+  thumbnail: Element<typeof Icon>,
+}): ReactNode {
+  return <Box aria-hidden>{cloneElement(thumbnail, { size: SIZE_ICON })}</Box>;
+}
+
 export function ToastIconThumbnail({ thumbnail }: { thumbnail: Element<typeof Icon> }): ReactNode {
```

Comment:
Can you add a prop to this? It looks like the exact same thing

Reply:
Done!